### PR TITLE
feat: web3.js templates

### PR DESCRIPTION
**Pull request by avkos**
# What :computer: 
This PR adds templates how to use web3js with zkSync

[CLI PR](https://github.com/matter-labs/zksync-cli/pull/121)
[Templates PR](https://github.com/matter-labs/zksync-scripting-templates/pull/4)
[web3-zksync-plugin](https://github.com/web3/web3-plugin-zksync)

# Why :hand:
* Reason why first thing was added to PR
* Reason why second thing was added to PR
* Reason why third thing was added to PR

# Evidence :camera:
Include screenshots, screen recordings, or `console` output here demonstrating that your changes work as intended

# Notes :memo:
* Any notes/thoughts that the reviewers should know prior to reviewing the code?

### CODE REVIEW

File: templates/nodejs/web3js/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+This is a [zkSync](https://zksync.io) + [ethers v6](https://docs.ethers.org/v6/) + [Node.js](https://nodejs.org/) project in TypeScript bootstrapped with [`zksync-cli`](https://github.com/matter-labs/zksync-cli)
```
Comment (luu-alex) [inline templates/nodejs/web3js/README.md]:
```suggestion
This is a [zkSync](https://zksync.io) + [web3js v4](https://docs.web3js.org/) + [Node.js](https://nodejs.org/) project in TypeScript bootstrapped with [`zksync-cli`](https://github.com/matter-labs/zksync-cli)
```

File: templates/nodejs/web3js/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+# Configuration
+
+By default your code will be run against zkSync Sepolia Testnet. If you wish to use a different network, you can change the `defaultChain` in `src/utils/chains.ts` file.
+
+Alternatively, you can use `getProvider` or `getWallet` with a custom `chain` parameter.
+
+# Learn more
+
+To learn more about [zkSync](https://zksync.io) or [ethers v6](https://docs.ethers.org/v6/), check out the following resources:
```
Comment (luu-alex) [inline templates/nodejs/web3js/README.md]:
```suggestion
To learn more about [zkSync](https://zksync.io) or [web3js v4](https://docs.web3js.org/), check out the following resources:
```

File: templates/nodejs/web3js/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- [zkSync Documentation](https://era.zksync.io/docs/dev)
+- [ethers v6 Documentation](https://docs.ethers.org/v6/)
```
Comment (luu-alex) [inline templates/nodejs/web3js/README.md]: update

**Comment by JackHamer09** [comment]
Thanks for your contribution @avkos 
Please make changes according to suggestions from @luu-alex (thanks tho!)
Also please make sure there is no package-lock file, since we allow users to use any of the package managers, therefore don't want to provide npm one by default

**Comment by avkos** [comment]
> Thanks for your contribution @avkos Please make changes according to suggestions from @luu-alex (thanks tho!) Also please make sure there is no package-lock file, since we allow users to use any of the package managers, therefore don't want to provide npm one by default

done. thanks

**Review by jdevcs — APPROVED** [review]

**Review by luu-alex — APPROVED** [review]

**Review by JackHamer09 — APPROVED** [review]

**Pull request merged**